lib/options.js

```diff
@@ -12,7 +12,8 @@ var knownOpts = {
     'loaderTemplatePath': path,
     'help': Boolean,
     'quiet': Boolean,
-    'version': Boolean
+    'version': Boolean,
+    'mustacheCompat': Boolean
```

Comment:
compat sounds fine. But this is specifically mustache compatibility mode for handlebars.